hack/secret-creator/create-plnsvc-secrets.sh

```diff
@@ -5,6 +5,7 @@ main() {
     create_namespace
     create_db_secret
     create_s3_secret
+    create_db_cert_secret_and_configmap
```

Comment:
ok so the analogous items here like the db and s3 secrets are pulled from vault and external secrets for stage and prod @avinal , which is why it is OK that this script creates them for the developer overlay

but there is no analogous vault / external secret item for this public cert

so even though things are not right as I type, even if we got them right, this would only work for the developer overlay and not stage/prod

I think the solution @scoheb outlined in the email you sent, where it adds the public cert to the system root CA when building the image, is the way to go, at least short term, since we have a Dockerfile we can update in the results downstream fork.

Now, when we move to the openshift pipelines operator version of results, we'll have to see if that same approach works ...i.e. is there a Dockerfile somewhere in the CPaaS pipeline that would could do the analogous thing for.

From @scoheb email:
- https://github.com/redhat-appstudio/managed-gitops/blob/main/Dockerfile#L49
- https://github.com/redhat-appstudio/segment-bridge/blob/22e23754649e150239d5e86f40dac7f99456d8e6/data/ca-trust/2022-IT-Root-CA.pem#L2

Reply:
/hold

while this script ^^  has progressed us along, I don't think that can be the final solution.

@enarha FYI

Reply:
I realized that such a script could be moved to a `Job` like what @Roming22  did for the chains secret over in https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/openshift-pipelines/chains-secrets-config.yaml#L46-L111

Now, I also know @enarha is hoping to rework how we do the chains secret (though I don't remember for sure if that rework included removing this Job)

Reply:
I am going through the mails and the threads. To summarize:
1. The RDS public cert can be added to the Dockerfile itself, this is how we are already doing
2. For development, we may want to use a Job, as we are doing in chains. 

If that is all, we can proceed with the discussion.